schedulePicker.py

```diff
@@ -4,7 +4,18 @@
 schedules = [][]
 
 def initCourses(datain):
-	
+	file1 = open(datain,"r");
+	classes = []
+	while true:
+		data = file1.readline()
+		if(data == ""):
+			break;
+		classes.append(data);
+	for i in range (0,len(classes)):
+		tmp = classes[i].split()
+		course = Course(tmp[0],tmp[1],tmp[2],tmp[3],tmp[4]);
```

Comment:
There are only four parameters for a Course. The fourth parameter is not a string, it is a dictionary of an array of tuples.